src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private static ColorSupplier supplier = new ColorSupplier();
+    protected String color;
+
+    public Figure() {
+        this.color = supplier.getRandomColor();
+    }
+
+    public abstract double getArea();
```

Comment:
let's create a separate interface AreaCalculator with abstract method getArea()

Reply:
I add this interface and delete this metod from figure class.